### DISCUSSION TITLE
Save only a single element of a webpage

### DISCUSSION THREAD

**Discussion by notDavid** [gildas-lormeau/SingleFile#1441] [Q&A] [answered]
Hi there,

noob question, i don't know anything about html... so bare with me;

Is it possible (possibly using Singlefile-cli) to only extract a single frame of a website?

I can see with uBlock the class (? i think) name is `##.speedread-pp.part`. Is it possible to only "extract' / "save" / "include" this frame?

Please advise,
Thank you!

**Comment by gildas-lormeau** [comment, marked as answer]
Hi,

It's possible via a userscript. It requires a little knowledge in JavaScript, see this similar question for example: https://github.com/gildas-lormeau/SingleFile/issues/1391
